recipes/ffmpeg/all/conanfile.py

```diff
@@ -176,24 +172,32 @@ def requirements(self):
     def validate(self):
         if self.options.with_ssl == "securetransport" and not tools.is_apple_os(self.settings.os):
             raise ConanInvalidConfiguration("securetransport is only available on Apple")
-        if self.settings.os == "Macos" and self.settings.arch == "armv8" and self.options.with_libvpx:
-            raise ConanInvalidConfiguration("libvpx doesn't support armv8 (yet)")
-        if self.settings.os in ["Linux", "FreeBSD"] and self.options.with_sdl:
-            raise ConanInvalidConfiguration("sdl is not supported by this recipe (yet)")
+
+    def build_requirements(self):
+        self.build_requires("yasm/1.3.0")
```

Comment:
I
```suggestion
        if self.settings.arch in ("x86", "x86_64"):
            self.build_requires("yasm/1.3.0")
```

Reply:
what is the warning? isn't yaml capable to generate ASM other from x86/x86_64?

Reply:
*yasm*  :wink: 
I don't think so.
[Their website](https://yasm.tortall.net/) only mentions *x86 and AMD64 instruction sets*

Reply:
yeah, and I remember [ffmpeg builds instructions](https://trac.ffmpeg.org/wiki/CompilationGuide/WinRT) mentioned to use [gas-preprocessor](https://github.com/libav/gas-preprocessor) for the ARM (maybe few other architectures). I assume it's not yet packaged in ConanCenter.
so probably, we have to build with `--disable-asm` for other architectures besides x86/x86_64 for a while.